tests/utils/test_remote_uploader.py

```diff
@@ -118,7 +118,9 @@ def _get_tmp_dir():
     with patch('composer.utils.object_store.utils.S3ObjectStore', DummyObjectStore):
         with patch('tempfile.TemporaryDirectory', _get_tmp_dir):
             with patch('composer.utils.remote_uploader.multiprocessing.get_context', lambda _: fork_context):
-                remote_uploader = RemoteUploader(remote_folder='S3://whatever/path',)
+                remote_uploader = RemoteUploader(
+                    remote_folder='S3://whatever/path',
+                )
```

Comment:
If you remove some trailing commas, yapf would inline some of these statements. For example here:
```suggestion
                remote_uploader = RemoteUploader(remote_folder='S3://whatever/path')
```
I know that `black` as an option for remove trailing commas if possible, not sure about `yapf`.

Reply:
Thanks, I'm open to that change, but just going to let yapf do what it does with current settings for now